### DISCUSSION TITLE
Synteny track not showing up

### DISCUSSION THREAD

**Discussion by ikumar2000** [GMOD/jbrowse-components#2012] [Q&A] [answered]
I created the PAF file using minimap2 and added it as a track. The two genomes have also been added. However, the synteny track doesn't show up in the browser (not even available for selection). This is all based on the super-quick guide. Is there something that I am missing?

**Comment by cmdcolin** [comment, marked as answer]
Hi there

If you are using the import form then you can do something like this
![jbrowse org_code_jb2_v1 3 0__config=test_data%2Fyeast_synteny%2Fconfig json session=local-z9Dg7EHrD (1)](https://user-images.githubusercontent.com/6511937/119537970-99cc3200-bd58-11eb-87e6-c9c1b5c3091f.png)
You can either provide a PAF file there, or if you added it via the command line, skip putting a PAF file there


If you provided a PAF file there, then you'll get this after you zoom out a few times (kind of starts out zoomed in, which may be something we change)
![jbrowse org_code_jb2_v1 3 0__config=test_data%2Fyeast_synteny%2Fconfig json session=local-z9Dg7EHrD](https://user-images.githubusercontent.com/6511937/119537977-9cc72280-bd58-11eb-83d6-716703098c81.png)


Note that the "synteny track" will be in the view menu-> "Open track selector" rather than in the view 1 or view 2 track selectors, so if you didn't supply a PAF file, this is where you would look
![jbrowse org_code_jb2_v1 3 0__config=test_data%2Fyeast_synteny%2Fconfig json session=local-z9Dg7EHrD (2)](https://user-images.githubusercontent.com/6511937/119538122-c7b17680-bd58-11eb-94c4-908f2d84b2d9.png)


Hope that helps :)

**Reply by cmdcolin** [reply]
Random troubleshooting considerations:

If your track selector is empty in the place I referred to, then make sure your synteny track is associated with the right "assembly names" (e.g. your synteny track should have "assemblyNames" in the config which matches the two assemblies you are comparing)

**Reply by ikumar2000** [reply]
Thanks.  

"synteny track" will be in the view menu-> "Open track selector"  did the trick. I have couple of follow up questions:


1. Can I visualize gff/gtf  or  bigwig files associated with each genome along with the syntenic view?
2. Can 3 genomes be simultaneously viewed?

Thanks

**Reply by cmdcolin** [reply]
1.  The dotplot doesn't have this but the linear mode does. You found that there is the "View menu"->"Open track selector". You should also be able to do "View menu"->"View 1"->"Open track selector" and see tracks on that particular organism or "View 2" -> "Open track selector" and see tracks on the other one

2. Currently we don't have 3 simultaneous. This actually existed in the first version of our synteny but the scope got reduced. We still want to implement this feature. If you want to see this feature come to life let us know...it is being tracked in this issue https://github.com/GMOD/jbrowse-components/issues/1134
